Pod/ILGClasses/ILGClasses.h

```diff
@@ -8,6 +8,8 @@
 
 #import <Foundation/Foundation.h>
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
Doesn't Swift bridge unannotated things to optionals by default?  Did they change that to force-unwrapping at some point?